server/fillRiak.js

```diff
@@ -0,0 +1,108 @@
+var fs = require('fs');
+var models = require('./models/riakModels.js');
+var Promise = require('bluebird');
+fs = Promise.promisifyAll(fs);
+
+
+
+async function makeFile(j) {
+    console.log(j);
+    var data = fs.readFileSync(`jsonData/file${j}.json`)
+    await (async () => {
+      data = JSON.parse(data);
+      var all = [];
+      for (var i in data) {
+        // console.log(i);
+        var obj = {
+          bucket: "restaurants",
+          key: data[i].id_name,
+          value: data[i]
+        }
+        var p = models.post(obj)
+        all.push(p);
+        // console.log(i);
+        if (i % 100 === 0) {
+          await Promise.all(all);
+          // console.log('hey good job', i);
+          all = [];
+        }
+      }
+    })()
+  }
+
+async function emptyBucket() {
+  var all = []
+  for (var i = 0; i <= 800000; i++) {
+    var obj = {
+      bucket: 'restaurants',
+      key: i.toString()
+    }
+    var p = models.delete(obj)
+    .then(() => {
+      // console.log('deleted');
+    })
+    .catch((err) => {
+      // console.log('uh oh', err)
+    })
+    all.push(p);
+  }
+  return(Promise.all(all))
+}
+
+// for (var j = 0; j <= 10; j++) {
+//   console.log('loopin!', j);
+//   makeFile(j);
+// }
+
+(async () => {
+  let date = new Date();
+  for (var k = 89; k <= 100; k++) {
```

Comment:
Is there any particular reason for 89-100?

Reply:
Leftover from when it stopped at 88.  Will fix